### PR TITLE
[ALLUXIO-2508]Improve javadoc in UnderFileStatus

### PR DESCRIPTION
**Pull request by maobaolong**
(https://alluxio.atlassian.net/browse/ALLUXIO-2508)

### CODE REVIEW

File: core/common/src/main/java/alluxio/underfs/UnderFileStatus.java [hunk 1]
```diff
@@ -14,7 +14,9 @@
 import javax.annotation.concurrent.NotThreadSafe;
 
 /**
- * Information about a file or a directory in the under file system.
+ * Information about a file or a directory in the under file system. Listing contents in a
+ * {@link UnderFileSystem}
+ * returns entries of this class.
```
Comment (calvinjia) [inline core/common/src/main/java/alluxio/underfs/UnderFileStatus.java]: Could you reduce this to two lines?

Comment (maobaolong) [inline core/common/src/main/java/alluxio/underfs/UnderFileStatus.java]: done

**Comment by maobaolong** [comment]
@calvinjia Thank you, I reduced to two line just now.

**Comment by calvinjia** [comment]
@maobaolong Thanks for the patch!

**Pull request merged**